shub/image/utils.py

```diff
@@ -31,10 +31,13 @@
 """
 
 
-def debug_log(msg):
+def is_verbose():
     ctx = click.get_current_context(True)
-    # FIXME remove check of debug param as it's deprecated
-    if ctx and (ctx.params.get('verbose') or ctx.params.get('debug')):
+    return ctx and ctx.params.get('verbose')
```

Comment:
Uhh you're right, there is a bug in the callback, it should `return value`. Can you add it?